server/src/main/java/io/crate/analyze/DropTableAnalyzer.java

```diff
@@ -77,8 +81,9 @@
         boolean maybeCorrupt = false;
         try {
             //noinspection unchecked
-            tableInfo = (T) schemas.resolveTableInfo(name, Operation.DROP, sessionContext.sessionUser(), sessionContext.searchPath());
+            tableInfo = (T) schemas.resolveTableInfo(name, sessionContext.sessionUser(), sessionContext.searchPath());
             tableName = tableInfo.ident();
+            validateDropOperation(tableInfo);
```

Comment:
Would it be possible to integrate this into `Operation.blockedRaiseException` instead of doing an additional validation in front of it? I think if there is a subscription we set a block. I assume that's why we already throw an error if there is a subscription.

Reply:
Would it make sense to include the information if a table is subscribed into the `TableInfo`? Then the information could be elegantly handled with `Operation.blockedRaiseException`.

Reply:
Yes/maybe. It should be somewhat encapsulated and within operation/tableInfo is probably a good place.

Might also be an option to flip things around and turn `supportedOperations` into `unsupportedOperations` -  where each unsupported operation can hold a reason with more details.